### DISCUSSION TITLE
Reading Dynamic Files

### DISCUSSION THREAD

**Discussion by Zeina-T** [pyinstaller/pyinstaller#7491] [Help] [answered]
Hi everyone,

I'm trying to package an application that will read files from the OS dynamically after packaging.
In summary, I have a folder with excel files that will get updated continuously. I want the application to be able to read from this folder in run time. 
The application runs fine when I include the files while packaging but the changes in these files obviously do not reflect in the application. 

Is there a way I can allow the packaged application to access other system files like this?

I tried to find an answer to this and couldn't, would really appreciate the help :)

**Comment by rokm** [comment, marked as answer]
You should be able to read any external files, as long as you properly reference them using their full, absolute path.

**Comment by Zeina-T** [comment]
Oh I think I tried everything _except_ that 🤯 💀 

Thanks a million!
